tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
     "skipLibCheck": true,
-    "strict": true,
+    "strict": false,
```

Comment:
curious... why did you set this to false?

Reply:
The event types on the `Searchbar` were still getting TS errors and it was taking me a while to fix them so I thought I'd leave it for this ticket https://newyorkpubliclibrary.atlassian.net/browse/DR-3082